truffle/pom.xml

```diff
@@ -23,6 +23,11 @@ DO NOT MODIFIY - GENERATED CODE
   </properties>
   <dependencies>
     <dependency>
+      <groupId>org.yaml</groupId>
+      <artifactId>snakeyaml</artifactId>
+      <version>1.14</version>
+    </dependency>
```

Comment:
I know you've asked in IRC, but we should find out how this will impact building the full dist.